lib/ansible/plugins/lookup/config.py

```diff
@@ -159,7 +156,7 @@ def run(self, terms, variables=None, **kwargs):
                     raise AnsibleLookupError('Unable to find setting %s' % term, orig_exc=e)
                 elif missing == 'warn':
                     self._display.warning('Skipping, did not find setting %s' % term)
-                elif missing == 'skip':
+                else:
                     pass  # this is not needed, but added to have all 3 options stated
```

Comment:
```suggestion
```
If you're going to get rid of stating the third option in the code, the comment no longer makes sense. I would either keep it or drop the branch entirely.

Reply:
Make sense.